frontend-react/src/pages/home/Sections/LiveMapSection.tsx

```diff
@@ -1,39 +1,66 @@
-import DOMPurify from "dompurify";
+import { Link } from "@trussworks/react-uswds";
 
 import { SectionProp } from "../HomeProps";
 import usamapsvg from "../../../content/usa_w_territories.svg";
-import { USLink } from "../../../components/USLink"; // in /content dir to get unique filename per build
+import { USLink } from "../../../components/USLink";
+import { Citation } from "../../../shared/Citation/Citation";
+
+import styles from "./LiveMapSection.module.scss";
 
 export default function LiveMapSection({ section }: { section: SectionProp }) {
-    let cleanDescriptionHtml = DOMPurify.sanitize(section!.description!);
     return (
-        <div className="tablet:margin-bottom-8">
+        <div>
             <h2
                 data-testid="heading"
-                className="font-sans-lg tablet:font-sans-xl margin-top-0 tablet:margin-bottom-0"
+                className="font-sans-xl margin-top-0 tablet:margin-bottom-0"
             >
                 {section.title}
             </h2>
-            <p
-                data-testid="summary"
-                className="usa-intro margin-top-1 text-base"
-            >
+            <p data-testid="summary" className="usa-intro margin-top-4">
                 {section.summary}
             </p>
-            <div data-testid="map">
+            <h3 data-testid="subTitle" className="font-sans-lg margin-top-4">
+                {section.subTitle}
+            </h3>
+            <div
+                data-testid="map"
+                className="desktop:grid-col-8 desktop:grid-offset-2 margin-y-8"
+            >
                 <USLink href="/product/where-were-live">
                     <img
                         src={usamapsvg}
                         title="USA with Territories (Heitordp, CC0, via Wikimedia Commons)"
                         alt="Map of states using ReportStream"
                     />
                 </USLink>
+                <div className="grid-row flex-justify-center">
+                    <ul className={styles.legend}>
+                        <li>
+                            <span className="bg-primary"></span>Connected
```

Comment:
Not sure if these empty spans were on purpose but if they are, just ignore this then. 😄

Reply:
Yes, they are used to create the legend squares